gulpfile.babel.js

```diff
@@ -1,36 +1,37 @@
-import babel from 'gulp-babel';
-import babelify from 'babelify';
-import browserify from 'browserify';
-import buffer from 'vinyl-buffer';
+import concat from 'gulp-concat';
 import ghPages from 'gulp-gh-pages';
 import gulp from 'gulp';
-import gutil from 'gulp-util';
-import ifElse from 'gulp-if-else';
 import print from 'gulp-print';
 import rimraf from 'rimraf';
-import source from 'vinyl-source-stream';
-import sourcemaps from 'gulp-sourcemaps';
 import uglify from 'gulp-uglify';
+import { rollup } from 'rollup-babel';
 
-function dist(minified) {
-  return browserify({entries: 'src/dist.js'})
-    .transform(babelify)
-    .bundle()
-    .pipe(source(minified ? 'field-kit.min.js' : 'field-kit.js'))
-    .pipe(buffer())
-    .pipe(sourcemaps.init({loadMaps: true}))
-      .pipe(ifElse(minified, uglify))
-      .on('error', gutil.log)
-    .pipe(sourcemaps.write('./'))
-    .pipe(gulp.dest('./dist'));
+function dist() {
+  return rollup({
+    entry: 'src/index.js',
+    babel: { loose: ['es6.spread', 'es6.parameters', 'es6.destructuring'] }
+  }).then(result =>
+    result.write({
+      dest: 'dist/field-kit.js',
+      format: 'umd',
+      moduleName: 'FieldKit'
```

Comment:
It's probably not very clear, but this string is what controls the global created in `dist/field-kit.js`, i.e. `global.FieldKit = factory()`.